ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -84,6 +85,17 @@ typedef NS_ENUM(NSInteger, ORKChoiceAnswerStyle) {
     ORKChoiceAnswerStyleMultipleChoice
 } ORK_ENUM_AVAILABLE;
 
+/// An enumeration of the format styles available for scale answers.
+typedef NS_ENUM(NSInteger, ORKNumberFormattingStyle) {
+    /// The default decimal style.
+    ORKNumberFormattingStyleDefault,
+    
+    /// Percent style.
+    ORKNumberFormattingStylePercent,
+    
+    /// Currency style.
+    ORKNumberFormattingStyleCurrency
```

Comment:
How about expose the number formatter as a read-only property? Users of the framework can then subclass and override the numberFormatter method to get specific desired behavior, including (for instance) adding a setter.